src/store/permissions/reducer.js

```diff
@@ -0,0 +1,39 @@
+import { applyReducerHash } from '@redhat-cloud-services/frontend-components-utilities/files/ReducerRegistry';
+import { ACTION_TYPES } from '../../constants';
+
+const loadWritePermissionsPending = (state) => ({
+    ...state,
+    loadingFailed: false,
+    loading: !insights.chrome.isProd,
+    writePermissions: insights.chrome.isProd
```

Comment:
I would set this to `undefined` while loading.

Reply:
^ basically the same answer